<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TView</class>
 <widget class="QMainWindow" name="TView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1352</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="TLines">
   <widget class="QLabel" name="l_Graphic">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>1001</width>
      <height>651</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Line">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>30</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="l_Line">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>N:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtRecalcular">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Recalcular Gráfico</string>
    </property>
   </widget>
   <widget class="QSlider" name="Slider">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>31</width>
      <height>281</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
    <property name="tickInterval">
     <number>5</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="gb_pulse">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Pulso</string>
    </property>
    <widget class="QRadioButton" name="rb_square">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Quadrado</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_gauss">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Gaussiano</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_lineNum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Quantidade de curvas</string>
    </property>
    <widget class="QRadioButton" name="rb_oneC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_twoC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_threeC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_param">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>161</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Parâmetros</string>
    </property>
    <widget class="QSplitter" name="splitter_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>141</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>N1:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>N2:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>N3:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>S1:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>S2:</string>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLineEdit" name="le_n1"/>
      <widget class="QLineEdit" name="le_n2"/>
      <widget class="QLineEdit" name="le_n3"/>
      <widget class="QLineEdit" name="le_s1"/>
      <widget class="QLineEdit" name="le_s2"/>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_zoom">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>360</y>
      <width>141</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Zoom</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>minX:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>59</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>maxX:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_minX">
     <property name="geometry">
      <rect>
       <x>52</x>
       <y>30</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_maxX">
     <property name="geometry">
      <rect>
       <x>52</x>
       <y>59</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_zApply">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Aplicar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_zRestore">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Restaurar</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
